uiplib/scheduler.py

```diff
@@ -19,11 +19,25 @@ def __init__(self, offline, pics_folder, timout, website, count):
         self.timeout = timout
         self.directory = pics_folder
         self.count = count
+
         if not offline:
-            fetch = Thread(target=self.initFetch)
-            # all child threads need to be daemons to die upon main thread exit
-            fetch.setDaemon(True)
-            fetch.start()
+            try:
+                #Init the threads
+                unsplashThread = Online_Fetch(1,self.website[0],self.directory,
```

Comment:
can we make this more generic? since you're calling the same function can we have as many threads as the websites user has specified? also do we need threads here? we can call them one by one sequentially as well?